scripts/analyze.py

```
@@ -0,0 +1,462 @@
+
+
+CHARSET_UPPER = set('ABCDEFGHIJKLMNOPQRSTUVWXYZ')
+CHARSET_LOWER = set('abcdefghijklmnopqrstuvwxyz')
+CHARSET_DIGIT = set('0123456789')
+CHARSET_SPECIAL = set('!@#$%^&*()_+-=[]{}|;:,.<>?/~`\'"\\ ')
+
+
+def load_entries(filepath: Path, encoding: str = 'utf-8') -> List[str]:
+    """Load non-empty, stripped entries from a wordlist file."""
+    try:
+        with open(filepath, 'r', encoding=encoding, errors='ignore') as f:
```

Ruff (0.15.17)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)